bugbug/similarity.py

```diff
@@ -626,6 +628,65 @@ def get_distance(self, query1, query2):
         raise NotImplementedError
 
 
+class ElasticSearchSimilarity(BaseSimilarity):
+    def __init__(self, cleanup_urls=True, nltk_tokenizer=False):
+        super().__init__(cleanup_urls=cleanup_urls, nltk_tokenizer=nltk_tokenizer)
+        self.elastic_search = Elasticsearch()
+        assert (
+            self.elastic_search.ping()
+        ), "Check if Elastic Search Server is running by visiting http://localhost:9200"
+
+    def get_text(self, bug):
```

Comment:
Same here, add the option in the base `get_text` to either get only the first comment or all comments.